src/uwtools/cli/experiment_manager.py

```diff
@@ -71,6 +66,6 @@ def parse_args(args: List[str]) -> Namespace:
     )
     parsed = parser.parse_args(args)
     if parsed.quiet and parsed.verbose:
-        print("Options --dry-run and --outfile may not be used together", file=sys.stderr)
+        print("Options --quiet and --verbose may not be used together", file=sys.stderr)
```

Comment:
bugfix

Reply:
At the time I was thinking that outputting nothing with `--dry-run` but requesting output with `--outfile` would be a user error. Since then we've moved to using the output file for more internal things, so replacing it with the standard q/v is the way  to go